.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Short description explaining the high-level reason for the new issue.
+
+## Current behavior
+
+-
+
+## Expected behavior
+
+-
+
+## Steps to replicate behavior
+
+1.
+
+
+## Screenshots
+
+__Current__
+
+
+__Expected__
```

Comment:
Agreed, I'll pull the both out and we'll be sure designers know to label what they're seeing vs what they expect. If it becomes an issue, we can always add them in later.